Makefile

```diff
@@ -407,3 +407,44 @@ consistent-crds-manifests-upstream:
 	$(Q)cd ./manifests-upstream/${OPERATOR_VERSION}/ && ln -srf ../../deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml \
 	servicebindingrequests.apps.openshift.io.crd.yaml
 
+## -- Target for merge to master dev release --
+.PHONY: merge-to-master-release
+## Make a dev release on every merge to master
+merge-to-master-release:
+	echo "${QUAY_TOKEN}" | docker login -u "redhat-developer+travis" --password-stdin quay.io
+	$(eval COMMIT_COUNT := $(shell git rev-list --count HEAD))
+	$(Q)operator-sdk build \
```

Comment:
Can we use the `/Dockerfile` to build the image? We don't use the `build/*` anywhere beyond `operator-sdk local`